ui/components/app/security-provider-banner-alert/security-provider-banner-alert.js

```diff
@@ -48,31 +49,53 @@ function SecurityProviderBannerAlert({
       )}
 
       {provider && (
-        <Text
-          marginTop={3}
-          display={Display.Flex}
-          alignItems={AlignItems.center}
-          color={Color.textAlternative}
-          variant={TextVariant.bodySm}
-        >
-          <Icon
-            className="disclosure__summary--icon"
-            color={IconColor.primaryDefault}
-            name={IconName.SecurityTick}
-            size={IconSize.Sm}
-            marginInlineEnd={1}
-          />
-          {t('securityProviderPoweredBy', [
-            <ButtonLink
-              key={`security-provider-button-link-${provider}`}
-              size={Size.inherit}
-              href={SECURITY_PROVIDER_CONFIG[provider].url}
-              externalLink
-            >
-              {t(SECURITY_PROVIDER_CONFIG[provider].tKeyName)}
-            </ButtonLink>,
-          ])}
-        </Text>
+        <>
+          <Text
+            marginTop={3}
+            display={Display.Flex}
+            alignItems={AlignItems.center}
+            color={Color.textAlternative}
+            variant={TextVariant.bodySm}
+          >
+            {t('somethingDoesntLookRight', [
+              <ButtonLink
+                key={`security-provider-button-supporturl-${provider}`}
+                size={Size.inherit}
+                href={SECURITY_PROVIDER_CONFIG[provider].supportUrl}
+                externalLink
+                onClick={onClickBlockaidSupport}
+              >
+                {t('contactUs')}
+              </ButtonLink>,
+            ])}
+          </Text>
```

Comment:
[suggestion] we could decouple the support URL from provider.

we can also use store the support URL in `metamask-extension/ui/helpers/constants/zendesk-url.js` and read from here. 

doing this we could end up with something like 

```
      <Text
        marginTop={3}
        display={Display.Flex}
        alignItems={AlignItems.center}
        color={Color.textAlternative}
        variant={TextVariant.bodySm}
      >
        {t('somethingDoesntLookRight', [
          <ButtonLink
            key={`security-provider-button-supporturl`}
            size={Size. inherit}
            href={ZENDESK_URLS.SUPPORT}
            externalLink
            onClick={onClickSupportLink}
          >
            {t('contactUs')}
          </ButtonLink>,
        ])}
      </Text>
          
      {provider && (
          <Text...
          unchanged code
          </Text>
     }
          
```